packages/angular/cli/src/commands/mcp/cli.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.dev/license
+ */
+
+import { StdioServerTransport } from '@modelcontextprotocol/sdk/server/stdio.js';
+import { Argv } from 'yargs';
+import { CommandModule, CommandModuleImplementation } from '../../command-builder/command-module';
+import { isTTY } from '../../utilities/tty';
+import { createMcpServer } from './mcp-server';
+
+const INTERACTIVE_MESSAGE = `
+To start using the Angular CLI MCP Server, add this configuration to your host:
+
+{
+  "mcpServers": {
+    "angular-cli": {
```

Comment:
Why not `ng mcp`?

Reply:
it's a host specific identifier that in some cases can be used in dot-notation.

Reply:
It seems to be a common pattern in the configuration.  Plus it bypasses global installation issues.  The command needs to be available globally.  We can adjust and iterate on the best approach based on feedback.